storage/database/db_manager.go

```diff
@@ -934,12 +939,31 @@ func (dbm *databaseManager) ReadHeadBlockHash() common.Hash {
 	return common.BytesToHash(data)
 }
 
+// Block Backup Hash operations.
+func (dbm *databaseManager) ReadHeadBlockBackupHash() common.Hash {
```

Comment:
How about combining this method with `ReadHeadBlockHash()`? So, we don't need to update the upper method.